.github/pull_request_template.md

```diff
@@ -1,14 +1,20 @@
 ## Pull Request Description
-
 [description here]
 
+### Related Pull Requests
+[related PRs from different repositories]
+
+### Related Issues
+[What issue(s) is the PR addressing]
+
 ## Checklist
 
 Not all may apply:
 
 - [ ] Tests (and test files) have been updated
 - [ ] Documentation has been updated
   - [ ] If related to resstock-estimation, checklist includes [data dictionary](https://github.com/NREL/resstock/tree/develop/resources/data/dictionary), [source report](https://github.com/NREL/resstock/tree/develop/project_national/resources/source_report.csv), [options saturation](https://github.com/NREL/resstock/tree/develop/project_national/resources/options_saturations.csv), [options_lookup](https://github.com/NREL/resstock/blob/develop/resources/options_lookup.tsv).
+  - [ ] Technical reference documentation (this box must be checked for the PR to get merged).
```

Comment:
Does this mean updating a latex file (?) today vs an rst version once it's migrated over?

Reply:
Right now it is the LaTex file, but once the internal reviews are done, then it will be the RST files. I will link to the docs when we have them and in RTD. It is more of a placeholder right now.